scipy/sparse/_csc.py

```diff
@@ -76,6 +76,9 @@ class csc_array(_cs_matrix):
       - slow row slicing operations (consider CSR)
       - changes to the sparsity structure are expensive (consider LIL or DOK)
 
+    Canonical format
+      - Within each column, indices are sorted by row.
+      - There are no duplicate entries.
```

Comment:
I personally like having these as separate bullet points, because the sorted-ness and unique-ness are separate concerns. That is, it's possible to have one without the other.